### ISSUE TITLE
[Project] Noise-aware neural networks for quantitative MRI

### ISSUE THREAD

**Issue by csparker** [Lewis-Kitchingman/VIC-HACK-2024#6]
**Title:** 
Noise-aware neural networks for quantitative MRI

 **Project Leader:** 
Christopher Parker, christopher.parker@ucl.ac.uk

**Project Description:** 
Neural networks are becoming an increasingly popular way of estimating parameters of biophysical models from MRI data. Yet, current approaches do not integrate knowledge of spatially-varying noise characteristics, which could influence the accuracy of parameter estimates. This project will explore different ways of estimating spatially-varying noise characteristics and of utilising them to build and test neural networks for biophysical modelling of MRI data. 

**Ideal Participant Characteristics:** 
- Python programming experience (including PyTorch, Tensorflow or Keras) would be useful but is not essential
- Interest in biophysical modelling / diffusion MRI / quantitative MRI / machine learning

**Resources:** 
- Preliminary work: https://arxiv.org/abs/2307.07072
- Python package: https://pypi.org/project/RicianLoss
- I will provide “realistic” simulated MRI datasets and other helpful code
- Laptop required!

**Tasks:** 
Some possible example tasks:
Task 1
Goal: Characterise spatially-varying noise 
Steps: Read in MRI data and use statistical techniques to build a map of the noise hyper-parameters
Task 2
Goal: Evaluate different techniques
Steps: Compare the accuracy of different noise characterisation techniques and how they perform on different MRI acquisitions 
Task 3
Goal: Build noise-aware neural networks
Steps: Integrate knowledge of the noise characteristics into neural network training and inference
